AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/LocationStatusReportsRoutes.java

```diff
@@ -40,6 +40,11 @@ public LocationStatusReportsRoutes(LocationReportsDao locationReportsDao, Locati
 
     @Override
     protected LocationStatusReport validateThenMap(LocationStatusReportApiModel apiModel) {
+
+        if (apiModel.getStatus() <= 0 || apiModel.getStatus() > 100){
```

Comment:
```suggestion
        if (apiModel.getStatus() < 0 || apiModel.getStatus() > 100){
```
0 is a valid value for status codes, using <= will make it invalid